src/GraphQL/TypeExtensions.cs

```diff
@@ -137,32 +140,34 @@ public static Type GetGraphTypeFromType(this Type type, bool isNullable = false,
                     if (mode == TypeMappingMode.UseBuiltInScalarMappings)
                     {
                         SchemaTypes.BuiltInScalarMappings.TryGetValue(type, out graphType);
+
+                        if (graphType == null)
+                        {
+                            if (type.IsEnum)
+                            {
+                                graphType = typeof(EnumerationGraphType<>).MakeGenericType(type);
```

Comment:
So we consider enums as built-in types (scalars) as well. Worth to add a note about it somewhere in xml comments or docs.